src/shared/dissect-image.c

```diff
@@ -2620,7 +2620,14 @@ static const char *const partition_designator_table[] = {
         [PARTITION_VAR] = "var",
 };
 
-int verity_dissect_and_mount(const char *src, const char *dest, const MountOptions *options) {
+int verity_dissect_and_mount(
+                const char *src,
+                const char *dest,
+                const MountOptions *options,
+                const char *host_os_release_id,
```

Comment:
maybe name this parameters `required_host_os_release_id` or so? i.e. these are the things